ros_ign_gazebo/CMakeLists.txt

```diff
@@ -0,0 +1,76 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(ros_ign_gazebo)
+
+find_package(catkin
+  REQUIRED
+  COMPONENTS
+    roscpp
+)
+
+# Citadel
+find_package(ignition-gazebo3 QUIET)
```

Comment:
Not necessary for this PR, but it may be convenient to have a macro for 
`find_package(ignition-citadel COMPONENTS gazebo transport msgs)` which populates the correct versions.

Reply:
That's an awesome idea :+1:

Reply:
Could you ticket an issue for it so we keep track? Thanks!

Reply:
https://bitbucket.org/ignitionrobotics/ign-cmake/issues/69/convenience-cmake-for-ignition-metapackage